draft-ietf-teas-yang-path-computation.txt

```diff
@@ -0,0 +1,4704 @@
+
+
+
+
+TEAS Working Group                                          I. Busi, Ed.
+Internet-Draft                                       Huawei Technologies
+Intended status: Standards Track                         S. Belotti, Ed.
+Expires: 28 July 2022                                              Nokia
+                                                     O. Gonzalez de Dios
+                                                              Telefonica
+                                                               A. Sharma
+                                                                  Google
+                                                                  Y. Shi
+                                                            China Unicom
+                                                           D. Ceccarelli
+                                                                Ericsson
+                                                         24 January 2022
+
+
+           A YANG Data Model for requesting path computation
+                draft-ietf-teas-yang-path-computation-17
+
+Abstract
+
+   There are scenarios, typically in a hierarchical Software-Defined
+   Networking (SDN) context, where the topology information provided by
```

Comment:
```suggestion
   Networking (SDN) architecture, where the topology information provided by
```

Reply:
Fixed in https://github.com/rvilalta/ietf-te-path-computation/pull/96/commits/fc6c5456c7cfaf1ec3ffa97477f75847d2fd428f

Co-Authored-By: sergio belotti <sergio.belotti@nokia.com>